src/types/observablemap.ts

```diff
@@ -306,6 +306,7 @@ export class ObservableMap<K = any, V = any>
                 other.forEach((value, key) => this.set(key, value))
             } else if (other !== null && other !== undefined)
                 fail("Cannot initialize map from " + other)
+            return
```

Comment:
NIT: I think dropping `return` here and on line 305 woudl achieve the same. (`fail` always throws)

Reply:
Agreed. This would be better as there is nothing need to be returned 😀